vnext/JSI/Shared/ChakraCoreRuntime.h

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifdef CHAKRACORE
+
+#include "ChakraCoreRuntimeArgs.h"
+#include "ChakraRuntime.h"
+
+#include "ChakraCoreDebugger.h"
+
+#include <memory>
+#include <string>
+
+namespace Microsoft::JSI {
+
+class ChakraCoreRuntime : public ChakraRuntime {
+ public:
+  ChakraCoreRuntime(const std::shared_ptr<ChakraCoreRuntimeArgs> &args) noexcept;
+  ~ChakraCoreRuntime() noexcept;
+
+#pragma region Functions_inherited_from_Runtime
```

Comment:
>Runtime [](start = 40, length = 7)

ChakraRuntime, no?

Reply:
Or should it be Jsi Runtime?

---
In reply to: [345518059](https://github.com/microsoft/react-native-windows/pull/3650#discussion_r345518059) [](ancestors = 345518059)